lib/styles/components/_button.scss

```diff
@@ -4,19 +4,18 @@
   font-size: $neeto-ui-text-sm;
   font-weight: $neeto-ui-font-medium;
   border-radius: $neeto-ui-rounded-sm;
-  line-height: 1.2;
+  line-height: 1;
   transition: $neeto-ui-transition;
   position: relative;
   border: none;
   cursor: pointer;
   padding: 0;
   user-select: none;
-  padding: 4px 8px;
```

Comment:
As per the [Design System](https://www.figma.com/file/Ur7M4IquUbQ57Jpyx1ciMH/Neeto-UI-Kit?node-id=174%3A1710), the smaller buttons have 28px height. When we remove the `min-height` and use the new `padding` this will be `30px`. Please confirm.

<img width="204" alt="Screenshot 2022-02-14 at 10 44 05 AM" src="https://user-images.githubusercontent.com/48869249/153813091-63ad55d7-683d-4a21-b95f-74713121ee0c.png">

Reply:
It is still 28px. Large button would be 31px instead of 32px.

<img width="953" alt="Screenshot 2022-02-14 at 8 08 01 PM" src="https://user-images.githubusercontent.com/24496302/153884420-963cef1a-700b-4212-93f5-a218c078a555.png">